docker-compose.yaml

```diff
@@ -7,9 +7,8 @@ services:
       context: ./nf_upf
       args:
         DEBUG_TOOLS: "false"
-    command: ./free5gc-upfd -f ../config/upfcfg1.yaml
     volumes:
-      - "./config/upfcfg1.yaml:/free5gc/config/upfcfg1.yaml"
+      - "./config/upfcfg1.yaml:/free5gc/config/upfcfg.yaml"
```

Comment:
I think different UPF might use different configuration file?

Reply:
Yes, but the mount location of the configuration file inside the container could be the same since each UPF lives in its own container. Having all UPFs find the configuration file under `/free5gc/config/upfcfg` won't be a problem.

Reply:
But the binding ip address, such as `pfcp:`, won't be the same.

Reply:
Yes of course, ip addresses won't be the same for `pfcp` of `gtpu` or `apns`. But, it's only the name of the configuration file inside the container. For example, if we do `./config/upfcfg1.yaml:/free5gc/config/upfcfg.yaml` and `./config/upfcfg2.yaml:/free5gc/config/upfcfg.yaml` for both `upf1` and `upf2`, this will not impact their configs, each one will have it's own configuration since we are mounting different config files. The only thing in common again is the mount point inside each container. My goal here is just to unify the naming of the configs inside the containers (less things to modify).

Reply:
Oh, I see, that's right